akvo/rsr/tests/views/test_account.py

```diff
@@ -505,7 +505,7 @@ def test_valid_json_data(self):
             'password2': 'passwdpasswdA1$',
         }
 
-        with patch('registration.models.RegistrationProfile.send_activation_email') as patched_send:
+        with patch('akvo.rsr.models.user.send_mail') as patched_send:
```

Comment:
I know it already existed like this, but maybe we should use the [testing email service](https://docs.djangoproject.com/en/1.11/topics/testing/tools/#email-services) the django provides here.